.github/workflows/cvmfs-ci.yml

```
@@ -162,6 +162,14 @@ jobs:
             ulimit -s unlimited
             text2workspace.py HiggsAnalysis/CombinedLimit/data/ci/datacard_RooMultiPdf.txt.gz  -o ws_RooMultiPdf.root
             combine -M MultiDimFit -m 125.38  --setParameters pdf_index_ggh=2 --freezeParameters MH --cminDefaultMinimizerStrategy 0 --X-rtd FAST_VERTICAL_MORPH --X-rtd MINIMIZER_freezeDisassociatedParams --X-rtd MINIMIZER_multiMin_maskChannels=2 --algo singles ws_RooMultiPdf.root
+            combine -M MultiDimFit -m 125.38 --freezeParameters MH --cminDefaultMinimizerStrategy 0 --X-rtd FAST_VERTICAL_MORPH --X-rtd MINIMIZER_freezeDisassociatedParams --X-rtd MINIMIZER_multiMin_maskChannels=2 --algo grid ws_RooMultiPdf.root --setParameterRanges r=-1,2
+            plot1DScan.py higgsCombineTest.MultiDimFit.mH125.38.root  -o scan_RooMultiPdf
+
+      - name: Upload scan_RooMultiPdf.pdf
+        uses: actions/upload-artifact@v4
+        with:
+          name: Scan for RooMultiPdf 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)